src/azure-cli/azure/cli/command_modules/containerapp/_up_utils.py

```diff
@@ -767,12 +766,6 @@ def _validate_up_args(cmd, source, image, repo, registry_server):
             "Cannot use --source and --repo togther. "
             "Can either deploy from a local directory or a Github repo"
         )
-    if repo and registry_server and "azurecr.io" in registry_server:
-        parsed = urlparse(registry_server)
-        registry_name = (parsed.netloc if parsed.scheme else parsed.path).split(".")[0]
```

Comment:
registry name also has to move?

Reply:
This part of check is due to in the following logic it use `register_name` as part of secret name:
`            registry_secret_name = "{server}-{user}".format(server=urlparse(registry_server).hostname.replace('.', ''), user=registry_user.lower())`
We don't need these check as more.

Reply:
Change the limit to 50 for ACR registry, which also the length limit for ACR:
![image](https://github.com/Azure/azure-cli/assets/46143499/56043635-fbca-4545-88b3-c91ea2712aad)